README.org

```diff
@@ -440,3 +445,20 @@ you get out of order matching!
 
 - Ivy has =ivy-restrict-to-matches=, bound to =S-SPC=, so you can get the
   effect of out of order matching without using =ivy--regex-ignore-order=.
+
+- It is possible to implement restriction to the current matches with
```

Comment:
What's the point of this?

Reply:
Well you document it here so one could also document that it is very easy to implement this in Orderless with default completion/Vertico/Mct/Icomplete. The code is here: https://github.com/minad/vertico/wiki#restrict-the-set-of-candidates. But really, there is no point besides that the candidates are locked in. But if we agree that such a behavior is not useful, then the Orderless documentation should probably clearly say that. On the other hand you also experimented with restricto, so maybe there is a point. ;)

Reply:
But I only document restriction in other systems as alternatives to orderless, as a way to get matching of several things in any order in those other systems. I'd prefer not to include this. It is a neat trick, though.

Reply:
It is okay. I had this lying around but I don't actually care about this and I don't use it. It is not more than a neat trick. Removed in https://github.com/oantolin/orderless/pull/112/commits/3fca979f6ee9d9973290e08eb38bfa4727284bc7.